front/app/components/SignUpIn/AuthProviders.tsx

```diff
@@ -257,6 +257,7 @@ const Data = adopt<DataProps>({
   franceconnectLoginEnabled: <GetFeatureFlag name="franceconnect_login" />,
   googleLoginEnabled: <GetFeatureFlag name="google_login" />,
   passwordLoginEnabled: <GetFeatureFlag name="password_login" />,
+  viennaLoginEnabled: <GetFeatureFlag name="vienna_login" />,
```

Comment:
@nTraum @luucvanderzee This should actually happen via the module somehow. (Just like the franceconnect stuff.)

Reply:
Edit: just read your conversation on Slack 👍